syncopy/io/utils.py

```diff
@@ -57,7 +57,7 @@ def hash_file(fname, bsize=65536):
     return hash.hexdigest()
 
 
-def cleanup(older_than=24, interactive=True, **kwargs):
```

Comment:
I have removed the kwargs here, as they are not needed and I had a typo (`olderthan` instead of `older_than`) in my first test script that was not reported because of the kwargs and caused me a bit of a headache.

Reply:
yes, good  idea.. there's too much `**kwargs` all over the place 😁